DALK.PL_ANALYZER/Views/Matches/Index.cshtml

```diff
@@ -3,41 +3,77 @@
 @{
     ViewBag.Title = "Meczyki";
 }
+ 
+    @{
+        string allMatchesClass = "layer";
+        if (Model.Count == 1)
+        {
+            allMatchesClass += " singleMatch";
+        } else {
+            allMatchesClass += " matches";
```

Comment:
Also this calculation should be put into ViewModel

Reply:
DONE.